project/includes/_functions_shared.sh

```diff
@@ -279,6 +279,13 @@ docker:status () {
   ./bin/docker-compose ${docker_compose_args} ps ${required_containers[*]}
 }
 
+############################
+# Restart all Docker containers
+############################
+docker_restart () {
+  docker_stop && docker_start
```

Comment:
@apedicdev , Thanks for the PR.

In this public version of docker-stack I've made the commands more consistent with expected norms.

"docker_stop" is now "docker:stop"
"docker_start" is now "docker:start"

Please would you rework your code so that your function is "docker:restart" and you call these functions.

Colin

Reply:
@cmtickle thanks is done